mobile/screens/HandlerInformationScreen.tsx

```diff
@@ -1,11 +1,86 @@
 import { StatusBar } from "expo-status-bar";
-import React from "react";
+import React, { useState } from "react";
 import { StyleSheet, Text, View } from "react-native";
+import DropDownPicker from "react-native-dropdown-picker";
+import { TextInput, TouchableOpacity } from "react-native-gesture-handler";
+import { Keyboard } from "react-native";
```

Comment:
What's the difference between the TouchableOpacity from react-native-gesture-handler and react-native? 

Noticed that TouchableOpacity from react-native-gesture-handler is a bit buggy on android but TouchableOpacity from react-native works fine